src/DynamoApplications/StartupUtils.cs

```diff
@@ -186,6 +186,33 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
             return model;
         }
 
+        /// <summary>
+        /// Attempts to import an assembly as a node library from a given file path.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <param name="path"></param>
+        internal static void ImportAssembly(DynamoModel model, string path)
```

Comment:
any thoughts on how to unify these duplicate functions now?

https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCLI/CommandLineRunner.cs#L96

I don't know why the DynamoCoreWPFCLI did not use the protected method, does it not inherit from the base class?

Reply:
I refactored the methods to use a single import function.